### PR TITLE
Fix getting-started.md with the correct import packages in main.go

### PR DESCRIPTION
**Pull request by fangn2**
This PR fixs the import packages section of [main.go](https://github.com/open-telemetry/opentelemetry-go/blob/main/website_docs/getting-started.md?plain=1#L278) in getting-started.md which is not synced with the [source code](https://github.com/open-telemetry/opentelemetry-go/blob/main/example/fib/main.go#L17). 

Following current getting-started guide as it misses few packages in the import, it will result in error.  
Check error message as blow:
dev-dsk-nhfang % go run .
\# fib
./main.go:27:21: undefined: resource
./main.go:28:10: undefined: resource
./main.go:29:3: undefined: resource
./main.go:30:3: undefined: resource
./main.go:34:4: undefined: attribute
./main.go:67:2: undefined: signal

### CODE REVIEW

**Review by hanyuancheung — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Slice attributes of `attribute` package are now comparable based on their value, not instance. (#3108 #3252)
 - Prometheus exporter will now cumulatively sum histogram buckets. (#3281)
 - Export the sum of each histogram datapoint uniquely with the `go.opentelemetry.io/otel/exporters/otlpmetric` exporters. (#3284, #3293)
+- Getting-started guide will now have the right list of import packages in main.go to run through the example successfully. (#3354)
```
Comment (dmathieu) [inline CHANGELOG.md:25]: I'm not sure website_docs fixes require a changelog entry.

Comment (pellared) [reply CHANGELOG.md]: https://github.com/open-telemetry/opentelemetry-go/pull/3354/commits/352dcd7bc260fba14178c5059f468a80ecb70c26

**Review by pellared — APPROVED** [review]

**Comment by pellared** [comment]
@fangn2 Thanks for the PR <3

**Review by dmathieu — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (fangn2) [reply CHANGELOG.md]: Thanks for changing that. I was not aware doc change doesn't need to go into changelog based on [CONTRIBUTING.md](https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md). Any written rules on this I can refer for future?

**Review by MrAlias — APPROVED** [review]

**Review by dashpole — APPROVED** [review]

**Pull request merged**